src/main/java/examples/pingpong/components/PingPongComponentsMain.java

```diff
@@ -112,7 +112,7 @@ private static void createInitialContracts(LedgerClient client, String sender, S
 
             // The wrapper object for commands with more details for the resulting transaction
             client.getCommandSubmissionClient().submit(
-                    String.format("Ping-%s-%d", sender, i),
+                    String.format("B-Ping-%s-%d", sender, i),
```

Comment:
```suggestion
                    String.format("Ping-%s-%d", sender, i),
```
Looks like a leftover from some debugging session. :wink: